src/components/MultiStoryContainer/styles.ts

```diff
@@ -8,7 +8,7 @@ const styles = StyleSheet.create({
     backgroundColor: Colors.black,
   },
   itemContainer: {
-    flex: 1,
+    height: Dimensions.get('window').height,
```

Comment:
@sagar-simformsolutions We should define height const and use that in itemContainer